.github/how_to_compile_gudhi_in_a_conda_env.md

```diff
@@ -0,0 +1,97 @@
+# Install a conda development environment to compile GUDHI
+
+## Install miniconda
+
+Download the [installer](https://docs.conda.io/en/latest/miniconda.html) required by your system and follow the [instructions](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+## Create a dedicated environment
+
+```bash
+conda install -c conda-forge mamba  # installation with mamba is faster
+conda create --name gudhi
+conda activate gudhi
+mamba install -c conda-forge python cmake doxygen eigen cgal-cpp
+```
+
+Some of the requirements are in the gudhi repository (please refer to
+[how to use github to contribute to gudhi](how_to_use_github_to_contribute_to_gudhi.md)).
+In the gudhi repository - let's call it `/workdir/gudhi` i.e. - once submodules are initialised:
+
+```bash
+pip install -r ext/gudhi-deploy/build-requirements.txt 
+pip install -r ext/gudhi-deploy/test-requirements.txt  # pytorch can be painful to install - not mandatory
+```
+
+## Compilation
+
+In order to compile all c++ utilities, examples, benchmarks, unitary tests, and python module:
+```bash
+cd /workdir/gudhi
+rm -rf build
+mkdir build
+cd build
+# To build all even examples and benchmarks
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=$CONDA_PREFIX -DWITH_GUDHI_EXAMPLE=ON -DWITH_GUDHI_BENCHMARK=ON ..
```

Comment:
So weird that `-DCMAKE_PREFIX_PATH=$CONDA_PREFIX` is needed (not your fault).

Reply:
https://cmake.org/cmake/help/latest/variable/CMAKE_SYSTEM_PREFIX_PATH.html says that `CMAKE_SYSTEM_PREFIX_PATH` on Unix contains
> ENV{CONDA_PREFIX} when using a conda compiler

But since `-DCMAKE_PREFIX_PATH=$CONDA_PREFIX` currently helps, whatever the reason, then it is indeed good to have it here.

Reply:
It was not working on my setup. Strange, because I have cmake 3.22, and this feature has been added on cmake 3.18.
As cmake minimal version is 3.5, I would prefer to keep it for the moment.